app/screens/home/search/search.tsx

```diff
@@ -156,7 +160,6 @@ const SearchScreen = ({teamId, teams}: Props) => {
     const handleCancelSearch = useCallback(() => {
         cancelRef.current = true;
         resetToInitial();
-        onSnap(0);
```

Comment:
so we dont need to snap it back to its origin? why?

Reply:
It's redundant. handleCancelSearch set the cancelRef.current to true.  And inside onFlatLayout, if cancelRef or clearRef is true, we onSnap(0) there too.